sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -8,12 +8,13 @@ DESIGN ?= FireSim
 # These guide chisel elaboration of the target design specified above.
 # See src/main/scala/SimConfigs.scala
 TARGET_CONFIG_PACKAGE ?= firesim.firesim
+#TARGET_CONFIG ?= FireSimNoMemPortConfig
 TARGET_CONFIG ?= FireSimRocketConfig
 
 # These guide chisel elaboration of simulation components by MIDAS, including models and widgets.
 # See src/main/scala/SimConfigs.scala
 PLATFORM_CONFIG_PACKAGE ?= firesim.firesim
-PLATFORM_CONFIG ?= BaseF1Config
+PLATFORM_CONFIG ?= BaseVitisConfig
```

Comment:
Probably also fine to revert and explicitly call out. This config is explicitly called in the Ci anyways